code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2094,6 +2094,40 @@
 	A.material_flags = applied_material_flags
 	A.set_custom_materials(metal_dat)
 
+/datum/reagent/obamium
+	name = "Obamium"
+	data = list("material"=null)
+	description = "A brown liquid made from a previous president. Can turn anything into his liking"
```

Comment:
"Can turn anything into his liking" is worded a bit awkwardly.

also, period.

Reply:
```suggestion
	description = "A brown liquid made from a previous president. Can turn anything that it touches into his face."
```